x-pack/plugins/security/server/authorization/reset_session_page.tsx

```diff
@@ -46,33 +46,12 @@ export function ResetSessionPage({
           <link href={path} rel="stylesheet" key={path} />
         ))}
         <Fonts url={uiPublicUrl} />
-        <link
-          rel="apple-touch-icon"
-          sizes="180x180"
-          href={`${uiPublicUrl}/favicons/apple-touch-icon.png`}
-        />
-        <link
-          rel="icon"
-          type="image/png"
-          sizes="32x32"
-          href={`${uiPublicUrl}/favicons/favicon-32x32.png`}
-        />
-        <link
-          rel="icon"
-          type="image/png"
-          sizes="16x16"
-          href={`${uiPublicUrl}/favicons/favicon-16x16.png`}
-        />
-        <link rel="manifest" href={`${uiPublicUrl}/favicons/manifest.json`} />
-        <link
-          rel="mask-icon"
-          color="#e8488b"
-          href={`${uiPublicUrl}/favicons/safari-pinned-tab.svg`}
-        />
-        <link rel="shortcut icon" href={`${uiPublicUrl}/favicons/favicon.ico`} />
+        {/* The alternate icon is a fallback for Safari which does not yet support SVG favicons */}
```

Comment:
This reset session page does not have the full complement of environment variables available to it, so I have excluded the environment check/favicon switch in favor of always showing the monochrome cluster version.

Reply:
@jportner thanks for the review and offer to assist. If you still have the time, I'd appreciate your contribution to this branch. Some time has passed since you commented, so I understand if you are busy with other things.